Dockerfile

```diff
@@ -1,32 +1,59 @@
-#FROM python:3.12.2
-FROM python:3.12.2-slim-bookworm
+FROM ubuntu
+# This will not work, since debian uses a slightly different apache2 config structure.
+# Current Dockerfile is meant to be used with the ubuntu apache2 setup.
+# FROM python:3.12.2-slim-bookworm
 
-ENV PYTHONUNBUFFERED 1
+ENV PYTHONUNBUFFERED=1
+ENV PYTHONDONTWRITEBYTECODE=1
 
-# Set the working directory in the container
-WORKDIR /app
+# Make port 80 and 443 available to the world outside this container.
+# 80 will be redirected to 443 using TLS through the apache.
+EXPOSE 80 443
+
+RUN apt-get update && apt-get install -y apt-utils vim curl apache2 apache2-utils python3 libapache2-mod-wsgi-py3 python3-pip python3-venv
+
+RUN a2enmod ssl
+RUN a2enmod rewrite
+
+RUN ln /usr/bin/python3 /usr/bin/python
 
 # Install poetry
 ENV POETRY_HOME=/opt/poetry
 RUN python -m venv $POETRY_HOME && $POETRY_HOME/bin/pip install poetry==1.8.2
 
-# Copy the current directory contents into the container at /app
-COPY . /app
+# Copy the current directory contents into the container
+COPY ./ /var/www/html/trustpoint/
+
+# Set the working directory in the container
+WORKDIR /var/www/html/trustpoint/
```

Comment:
Maybe for the beta it is not that important but bevor our first realease we should think about data persistency.
An outsourced db or at least a volume in a docker context so we do not lose our settings/data if the container dies or we restart it.

Reply:
can't we just make the sqlite3 file persistent for now?